storage/replica.go

```diff
@@ -202,7 +203,7 @@ type Replica struct {
 	// Last index applied to the state machine. Updated atomically.
 	appliedIndex uint64
 	configHashes map[int][]byte // Config map sha256 hashes @ last gossip
-	systemDBHash []byte         // sha256 hash of the system config @ last gossip
+	systemDBHash []byte         // sha1 hash of the system config @ last gossip
```

Comment:
Why make this change? We certainly don't need secure cryptographic hashing for this, but sha1 is eventually going to be replaced by sha2 for most uses. Why import a different algorithm for this purpose?